integr8ly/installation
# INTLY-2752 Update msbroker to 0.0.7

## Additional Information
https://issues.jboss.org/browse/INTLY-2752

## Verification Steps

Install

1. Verify install completes
2. Verify the new clusterrole & clusterrolebinding exists `oc get clusterrole managed-service-role-for-syndesis-operator` `oc get clusterrolebinding default-cluster-account-managed-service-for-syndesis-operator`

Upgrade

1. Verify upgrade from 1.4.1 succeeds
2. Verify the new clusterrole & clusterrolebinding exists `oc get clusterrole managed-service-role-for-syndesis-operator` `oc get clusterrolebinding default-cluster-account-managed-service-for-syndesis-operator`

## Is an upgrade task required and are there additional steps needed to test this?

Yes. Included.
